packages/relay/tests/lib/sdkClient.spec.ts

```diff
@@ -174,56 +174,4 @@ describe('SdkClient', async function () {
       sinon.assert.calledOnce(convertGasPriceToTinyBarsStub);
     });
   });
-
-  describe('deleteFile', () => {
```

Comment:
Are we deleting them entirely ?

Reply:
Not sure, we should actually now that I have double checked. Will take a second look